src/skmultiflow/meta/learn_nse.py

```diff
@@ -0,0 +1,286 @@
+            estimator.partial_fit(X, y, classes=classes)
+
+    def partial_fit(self, X, y=None, classes=None, weight=None):
+        """
+        Partially fits the model, based on the X and y matrix.
+        Parameters
+        ----------
+        X: Numpy.ndarray of shape (n_samples, n_features)
+            Features matrix used for partially updating the model.
+        y: Array-like
+            An array-like of all the class labels for the samples in X.
+        classes: list
+            List of all existing classes. This is an optional parameter, except
+            for the first partial_fit call, when it becomes obligatory.
+        Raises
+        ------
+            RuntimeError:
+                A RuntimeError is raised if the 'classes' parameter is not
+                passed in the first partial_fit call, or if they are passed in further
+                calls but differ from the initial classes list passed.
+                A RuntimeError is raised if the base_estimator is too weak. In other word,
+                it has too low accuracy on the dataset.
+        Returns
+        -------
+        LearnNSE
+            self
+        """
+        N, _ = X.shape
+        if self.classes is None:
+            if classes is None:
+                raise RuntimeError("Should pass the classes in the first partial_fit call")
+            else:
+                self.classes = classes
+
+        for i in range(N):
+            self.X_batch.append(X[i])
+            self.y_batch.append(y[i])
+            mt = len(self.y_batch)
+
+            if mt == self.period:
+                self.X_batch = np.array(self.X_batch)
+                self.y_batch = np.array(self.y_batch)
+
+                classifier = cp.deepcopy(self.base_estimator)
+
+                if len(self.ensemble) > 0:
+                    # Compute the error of the existing ensemble on the new data
+                    votes = self.predict(self.X_batch)
+
+                    et = np.sum(votes != self.y_batch) / mt
+
+                    # Update and normalize instance weights
+                    self.instance_weights = np.ones(mt) / mt
+                    self.instance_weights[votes == self.y_batch] = et / mt
+
+                    # normalize instance weights (distribution)
+                    self.instance_weights = self.instance_weights / np.sum(self.instance_weights)
+
+                    # Train base classifier with Dt
+                    self.__fit(classifier, self.X_batch, self.y_batch, sample_weight=self.instance_weights,
+                               classes=self.classes)
+                    # classifier.fit(self.X_batch, self.y_batch, sample_weight=self.instanceWeights)
+
+                else:
+                    # First run! train the classifier on the instances with the same weight
+                    self.instance_weights = np.ones(mt) / mt
+
+                    # classifier.fit(self.X_batch, self.y_batch, sample_weight=self.instanceWeights)
```

Comment:
Save comment as above, weights are not used to train the model `sample_weight=None`.

Also, please remove commented code.